feature-foryou/src/main/java/com/google/samples/apps/nowinandroid/feature/foryou/ForYouScreen.kt

```diff
@@ -203,8 +205,13 @@ fun ForYouScreen(
                     loadingContentDescription = R.string.for_you_loading
                 )
 
-                item(span = { GridItemSpan(maxLineSpan) }) {
```

Comment:
Ah sorry, this wasn't a clean merge. We changed to use a `LazyGrid` view here, and therefore we want the spacing items to fill up the entire span (all columns, if there are more than one).

That ensures that this spacing is applied correctly, as opposed to being an item at the end of a row after an item that doesn't fill up all of the space.

Let's add the `span = { GridItemSpan(maxLineSpan) }` back in to the `item`s. We also probably could have just one `item` instead, and have a `Column` of the two `Spacer`s.

Reply:
Done ! Can you check ?